index.js

```diff
@@ -219,9 +219,20 @@ function tsLookup({dependency, filename, tsConfig}) {
 
   const host = ts.createCompilerHost({});
   debug('with options: ', options);
-  const resolvedModule = ts.resolveModuleName(dependency, filename, options, host).resolvedModule;
-  debug('ts resolved module: ', resolvedModule);
-  const result = resolvedModule ? resolvedModule.resolvedFileName : '';
+
+  const namedModule = ts.resolveModuleName(dependency, filename, options, host);
+  let result = '';
+
+  if (namedModule.resolvedModule) {
+    result = namedModule.resolvedModule.resolvedFileName;
+  } else {
+    const suffix = '.d.ts';
```

Comment:
What does the .d.ts extension refer to? And why would the not found files have that extension?